src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -72,7 +72,8 @@
               i18n-title
               i18n-description
               class="toolbar-action"
-              placement="bottom">
+              placement="bottom"
+              *ngIf="fetchData.observers.length > 0">
```

Comment:
Just to be on safer side.
```suggestion
              *ngIf="fetchData?.observers?.length > 0">
```

Reply:
done.